### DISCUSSION TITLE
[Projects Beta] Save the expanded/collapsed status group states

### DISCUSSION THREAD

**Discussion by ketanmistry** [community/community#8566] [Projects and Issues]
It would be good if the expanded/collapsed state of the status groups is remembered. 

For example, I use the "in progress" group most of the time, and if the groups are before this are quite long, it results in scrolling down all the time when revisiting the page.

![CleanShot 2021-12-09 at 07 30 41](https://user-images.githubusercontent.com/6877442/145280626-823bad5d-03a3-4b3d-95ba-082f8bfee312.png)

**Comment by mikermcneil** [comment]
Related: https://github.com/github/feedback/discussions/10080
